tests/Unit/Core/Module/ModuleRepositoryTest.php

```diff
@@ -50,8 +50,17 @@ class ModuleRepositoryTest extends TestCase
         'demo',
     ];
 
+    private const ACTIVED_MODULES = [
```

Comment:
```suggestion
    private const ACTIVE_MODULES = [
```

I think `actived` doesn't make much sense here, it would be `active` or `activated` ;)